test/runnable/testargtypes.d

```diff
@@ -30,52 +30,59 @@ void chkPair(T,U,V)()
     chkArgTypes!(S, V)();
 }
 
-version (X86_64)
+version (Posix) version (X86_64) version = Posix_X86_64;
+
+version (Posix_X86_64)
```

Comment:
Wouldn't this mean that windows x86_64 abi is no longer being tested?

Reply:
These tests incl. Win64 were in vain/useless/confusing. They tested the SysV argtypes stuff on Windows, which is as wrong as it gets. But this should be discussed in #8837; I don't know why that commit gets a lot more feedback here than in its actual smaller PR.